packages/bloc_test/lib/src/bloc_test.dart

```diff
@@ -170,6 +171,271 @@ void blocTest<B extends BlocBase<State>, State>(
   );
 }
 
+/// Creates a new `bloc`-specific test case with the given [description].
```

Comment:
I suppose we could also add some sort of example using `fakeAsync.elapse` which is kind of the main purpose of this new method, wdyt?

Reply:
We might wanna update the README.md docs as well and any other sort of documentation bloc_test might have